AmazonChimeSDK/AmazonChimeSDK/internal/ingestion/AnyCodable.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  AnyCodable.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public struct AnyCodable: Codable {
```

Comment:
Were you able to check it works for Objective C as well?

Reply:
What happens when value is `nil`? Does it pass nil or just not included in the Json?

Reply:
> Were you able to check it works for Objective C as well?

No, I doesn't. I was planning to make it support Obj-C, but I didn't find any consumer classes public facing(even though they are `public`, there is no public interface using them).

> What happens when value is `nil`? Does it pass nil or just not included in the Json?

There is a helper initializer at line 81 handles nil.

Reply:
So `DefaultEventReporter` can be injectable and people can pass custom one as well. That's why these are public.

Reply:
I don't see anywhere in `DefaultEventReporter` could link to AnyCodable. Could you show me?